tubular/nominal.py

```diff
@@ -747,6 +747,12 @@ def fit(self, X, y):
 
         return self
 
+    def map_imputation_values(self, X):
```

Comment:
Please can this have a docstring

Reply:
de265f2a3c073d4d7e4ed5ae70e5237ee2205cbb